themes/default/layouts/page/get-started.html

```diff
@@ -73,7 +48,7 @@ <h3><span class="text-2xl">Start learning</span></h4>
             <div class="w-full lg:w-1/2 px-4 py-2 lg:p-8 lg:px-4">
                 <div class="lg:shadow-2xl pl-12 pr-12 pb-12 pt-8 g:pb-12 rounded-xl bg-white">
                     <h3><span class="text-2xl">Explore</span></h4>
-                    <p class="mb-10">The optional Pulumi Console UI manages deployment state and provides collaboration tools. Sign in to explore!</p>
+                    <p class="mb-10">The optional Pulumi Console manages deployment state and provides collaboration tools. Sign in to explore!</p>
```

Comment:
Sure, but as you might imagine, we've discussed this before, and we ultimately settled on "Pulumi Console" and "Pulumi Service", which is how you'll see these described pretty much everywhere. It'd definitely be a bigger discussion to change how we refer to these.

Reply:
im trying to manage feedback form multiple folks, so its much easier to have them as suggestions instead of actual changes to help me manage the convos.